utilities/EluvioCli.js

```diff
@@ -744,6 +899,20 @@ yargs(hideBin(process.argv))
     }
   )
 
+  .command(
+    "account_set_tenant_contract <tenant>",
```

Comment:
Let's keep one of these - `account_set_tenant` and set both the tenant admins group and tenant contract ID.  We never need to set these independently